src/test/java/de/retest/recheck/review/ignore/MatcherFilterLoaderTest.java

```diff
@@ -70,9 +70,9 @@ void load_should_produce_correct_ignore() {
 	}
 
 	@Test
-	void load_with_invalid_matcher_should_not_load_parent() throws Exception {
+	void load_with_generic_matcher_should_load_correctly() throws Exception {
```

Comment:
`load_with_invalid_matcher_should_not_load_parent()` -> so there is no longer any invalid matcher possible with this new implementation?

Reply:
Good point, I'll add a test